homework/src/App.js

```diff
@@ -17,12 +17,25 @@ class App {
     // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
 
     const root = document.getElementById('root');
+
+    const header = Util.createAndAppend('header', root, { 'html': '<h3> HYF Repositories <h3>' });
```

Comment:
See the feedback that @JawharBairakdar already gave you in week 2 but somehow didn't make it in your week 3 homework.

Reply:
i'm not sure if i can commit here ! but thanks for your feedback ,  and i've just finished updating my homework  for week 2 and 3 , i'm late but i'm doing my best thanks alot .